sdk/core/loader/boot.cc

```diff
@@ -1214,7 +1214,19 @@ extern "C" SchedulerEntryInfo loader_entry_point(const ImgHdr &imgHdr,
 	// accesses these in the revoker is very small and amenable to auditing
 	// (the only memory accesses are a load and a store back at the same
 	// location, with interrupts disabled, to trigger the load barrier).
-	auto scaryCapabilities = build<void *, Root::Type::RWStoreL>(
+	//
+	// We use imprecise set-bounds operations here because we need to ensure
+	// that the regions are completely scanned and scanning slightly more is
+	// not a problem unless the revoker is compromised.  The software revoker
+	// already has a terrifying set of rights, so this doesn't really make
+	// things worse and is a nother good reason to use a hardware revoker.
+	// Given that hardware revokers are lower power, faster, and more secure,
+	// there's little reason for the software revoker to be used for anything
+	// other than testing.
+	auto scaryCapabilities = build<void *,
+	                               Root::Type::RWStoreL,
+	                               Root::Permissions<Root::Type::RWStoreL>,
+	                               false>(
```

Comment:
Since we aren't doing the type-level enumeration thing, perhaps
```suggestion
	                               /* Precise: */ false>(
```